qa/workunits/rgw/test_rgw_d4n.py

```diff
@@ -148,6 +149,9 @@ def get_body(response):
     return got
 
 def test_small_object(r, client, s3):
+    shutil.rmtree('/tmp/rgw_d4n_datacache/')
```

Comment:
@samarahu : can you explain why this is needed?

Reply:
I was using it while testing locally, but it isn't necessary. I'll remove it and update the commit.